packages/devtools_app/lib/src/shared/vm_flags.dart

```diff
@@ -21,40 +21,40 @@ const profiler = 'profiler';
 const profilePeriod = 'profile_period';
 
 class VmFlagManager extends Disposer {
-  VmServiceWrapper get service => _service;
-  VmServiceWrapper _service;
+  VmServiceWrapper? get service => _service;
+  VmServiceWrapper? _service;
 
   ValueListenable get flags => _flags;
-  final _flags = ValueNotifier<FlagList>(null);
+  final _flags = ValueNotifier<FlagList?>(null);
 
-  final _flagNotifiers = <String, ValueNotifier<Flag>>{};
+  final _flagNotifiers = <String?, ValueNotifier<Flag?>>{};
```

Comment:
Flag does not need to be non-nullable

Reply:
you mean nullable?

Reply:
yes, sorry. This should be non-nullable is what I meant.

Reply:
done